subprojects/gdk_rust/gdk_common/src/network.rs

```diff
@@ -26,6 +26,8 @@ pub struct Network {
     pub ct_exponent: Option<i32>,
     pub ct_min_value: Option<u64>,
     pub spv_enabled: Option<bool>,
+    pub spv_cross_validation: Option<bool>,
```

Comment:
This unfortunately will cause the root dir for cache and store to change, thus the old store and its memos and settings will be ignored.

        let wallet_desc = format!("{}{:?}", xpub, self.network);
        let wallet_id = hex::encode(sha256::Hash::hash(wallet_desc.as_bytes()));
        ...
        let mut path: PathBuf = self.data_root.as_str().into();
        if !path.exists() {
            std::fs::create_dir_all(&path)?;
        }
        path.push(wallet_id);
        info!("Store root path: {:?}", path);

I think these new elements should be ignored when computing `wallet_desc`. Which is quite ugly, but if we want to limit the scope of this MR, we might need to accept a similar solution.

Reply:
Good catch! Addressed by c73f42779648e328b8cfa95a601ef085f43d6507.